cmd/podman/compose.go

```diff
@@ -202,7 +202,7 @@ func composeProviderExec(args []string, stdout io.Writer, stderr io.Writer, warn
 	logrus.Debugf("Executing compose provider (%s %s) with additional env %s", provider, strings.Join(args, " "), strings.Join(env, " "))
 
 	if warn {
-		fmt.Fprint(os.Stderr, underline(fmt.Sprintf(">>>> Executing external compose provider %q. Please refer to the documentation for details. <<<<\n\n", provider)))
+		fmt.Fprint(os.Stderr, underline(fmt.Sprintf(">>>> Executing external compose provider %q. Please see podman-compose(1) for how to disable this message. <<<<\n\n", provider)))
```

Comment:
Podman will work with docker-compose also?

Reply:
Beats me, but according to https://linuxconfig.org/how-to-use-docker-compose-with-podman-on-linux there seems to be some kind of use case for it I guess?

Reply:
`podman-compose(1)` is the proper syntax for our man page. I am not sure if the actual podman-compose program ships a man page but likely not otherwise we would have heard about conflicts by now.

Reply:
I'm no Go expert (by far, at all) but it seems to me fmt.Fprint can only output unformatted text. Thus I did not change this from the original commit. It seems fmt.Fprintf could print formatted text like `podman-compose(1)` or such. Please advise if necessary.